BazelExtensions/workspace.bzl

```diff
@@ -279,5 +287,6 @@ def new_pod_repository(name,
         trace=trace,
         enable_modules=enable_modules,
         generate_module_map=generate_module_map,
-        header_visibility=header_visibility
+        header_visibility=header_visibility,
+        extraction_handler=extraction_handler
```

Comment:
(trailing comma plz 🙌 )

Reply:
Wasn't sure if skylark allowed it so I opted to just stick with the existing convention.

Reply:
So I thought about this a bit and am going to keep this and the above as is... My reasoning is that if this is going to be merged to the upstream master it should try and follow their conventions.

Reply:
That's a really good point, good call

![good-idea](https://user-images.githubusercontent.com/463696/58209300-9e5a3d80-7c9b-11e9-83cb-4ec17d070fb7.gif)